src/API.CRUD/ActionFilters/ValidateModelAttribute.cs

```diff
@@ -10,7 +11,17 @@ public override void OnActionExecuting(ActionExecutingContext context)
         {
             if (!context.ModelState.IsValid)
             {
-                context.Result = new BadRequestObjectResult(context.ModelState);
+                var modelStateErrors = new Dictionary<string, string>();

```

Comment:
For discussion tomorrow/whenever: can you show me the difference (visually, e.g in the browser) between the old/new code? Reason being, i thought the code on the LHS already did this.

Reply:
Happy with this. Discussion on `ApiModel` schema.